.github/CODEOWNERS

```diff
@@ -69,9 +69,11 @@
 # Canvas
 /x-pack/legacy/plugins/canvas/  @elastic/kibana-canvas
 
-# Logs & Metrics UI
+# Observability UIs
 /x-pack/legacy/plugins/infra/ @elastic/logs-metrics-ui
+/x-pack/plugins/infra/ @elastic/logs-metrics-ui
 /x-pack/legacy/plugins/integrations_manager/ @elastic/epm
+/x-pack/plugins/observability/ @elastic/logs-metrics-ui @elastic/apm-ui @elastic/uptime @elastic/epm
```

Comment:
`@elastic/epm` should be `@elastic/ingest` (we're in the process of cleaning up the duplicate team labels after the reorg 🙂, but `ingest` should be the one to go with)

Reply:
Yeah but @jen-huang can you let me know what the team name will be next week too, so I can pre-emptively update to that? 🥁 🎤